service-info/README.md

```diff
@@ -0,0 +1,55 @@
+# GA4GH Service Info Types
+
+This file details how to add values to the list of Service Info type values for the GA4GH API.
+
+## Background
+
+To enable servers using the suite of GA4GH APIs to be used in a networks, the Service Info endpoint was created. A centrally maintained list of the type values for GA4GH APIs was requested, which was to be managed by the GA4GH TASC Force. See [here for the main Service Info repository](https://github.com/ga4gh-discovery/ga4gh-service-info).
+
+## Using These Files
+
+There are two accompanying files here which should have the same information in both formats.
```

Comment:
What's the reason for this duplication?

Reply:
I believe was this to make it easier to process computationally for different resources. @jb-adams can you offer further elaboration?

Reply:
I don't know if we agreed on a single format (json or tsv) by the time I made the PR. Now that we've agreed to use json I'll delete the tsv